exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -67,8 +67,8 @@ var (
 )
 
 // MakeSegmentDocumentString converts an OpenTelemetry Span to an X-Ray Segment and then serialzies to JSON
-func MakeSegmentDocumentString(span ptrace.Span, resource pcommon.Resource, indexedAttrs []string, indexAllAttrs bool) (string, error) {
-	segment, err := MakeSegment(span, resource, indexedAttrs, indexAllAttrs)
+func MakeSegmentDocumentString(span ptrace.Span, resource pcommon.Resource, indexedAttrs []string, indexAllAttrs bool, logGroupNames []string) (string, error) {
```

Comment:
@mx-psi
Bumping this question.

Reply:
Hey, sorry, missed this.

> is there a risk of a breaking change with changing the signature of this as a public method? I don't think so since it's part of the collector rather than an SDK, but want to confirm

This is part of an internal package, so it is not part of the public API and thus there is no risk.

In general, we do care about the Go API of the Collector: we aim for Collector components to be eventually stable in both their usage as part of a Collector binary and as a library, but in this case this is fine.